concepts/05 Widgets/DataGrid/15 Columns/35 Hide a Column Using the API.md

```diff
@@ -26,6 +26,78 @@ A column is considered hidden when its [visible](/api-reference/_hidden/GridBase
         ],
         // ...
     })
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ... >
+            <DxColumn
+                data-field="Email"
+                :visible.sync="isEmailVisible"
+            />
+        </DxDataGrid>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxDataGrid, {
+        DxColumn
+    } from 'devextreme-vue/data-grid';
+
+    export default {
+        components: {
+            DxDataGrid,
+            DxColumn
+        },
+        data() {
+            return() {
+                isEmailVisible: true
+            }
+        },
+        methods: {
+            hideEmails() {
+                this.isEmailVisible = false;
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DataGrid from 'devextreme-react/data-grid';
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+            this.dataGridRef = React.createRef();
+
+            this.hideEmails = () => {
+                this.dataGrid.columnOption('Email', 'visible', false);
```

Comment:
This approach is not good for React. Let's use React State to change the visible option.

You can check my examples in the Filtering guides (e.g. https://js.devexpress.com/Documentation/Guide/Widgets/DataGrid/Filtering_and_Searching/#Filter_Row)

Reply:
I decided to use this approach because it is universal if you declare columns in the markup as well as if you use `customizeColumns`.

Reply:
Ah, I got your idea. Then, it's strange that in Vue we demonstrated two-way binding.
:visible.sync="isEmailVisible"
At least we need to change Vue for consistency. Or (what is better) we can demonstrate  two possible solutions (bindings and our API)  for Angular, React, and Vue :)

Reply:
I've updated the code. It uses two-way binding now.